src/pybind/mgr/rgw/module.py

```diff
@@ -123,7 +153,7 @@ def _cmd_rgw_admin(self, params: Sequence[str]):
         return HandleCommandResult(retval=returncode, stdout=out, stderr=err)
 
     @CLICommand('rgw realm bootstrap', perm='rw')
-    @check_orchestrator()
+    @check_orchestrator
```

Comment:
is there maybe a world where we would want to allow users to do this without having cephadm running so they can streamline the realm creation but still do the daemon deployment on their own or with another tool?

Reply:
Well if that's a possibility then probably we can give that option. However I'd address it as separate PR because that will need more testing. I opened https://tracker.ceph.com/issues/58506 to track the change.